### DISCUSSION TITLE
Probblème s'installation de LollMs

### DISCUSSION THREAD

**Discussion by zokalinga** [ParisNeo/lollms-webui#556] [General]
Voici un modèle de courriel adapté à votre situation pour expliquer le problème d'installation de Lollms :

Objet : Problème d'installation de Lollms - Fichier Miniconda introuvable

Bonjour,

Je vous contacte car je rencontre des difficultés pour installer Lollms sur mon ordinateur Windows. Malgré plusieurs tentatives, je n'arrive pas à finaliser l'installation en raison des problèmes suivants :

1. L'installateur de Lollms ne parvient pas à trouver le fichier "Miniconda3-latest-Windows-x86_64.exe". J'obtiens le message d'erreur : "Windows ne trouve pas 'Miniconda3-latest-Windows-x86_64.exe'. Vérifiez que vous avez entré le nom correct, puis réessayez."

2. J'ai également remarqué un avertissement concernant la longueur du chemin d'installation, indiquant que l'installation pourrait échouer si le chemin est trop long.

J'ai essayé de placer le fichier Miniconda dans le dossier spécifié (C:\Users\EVA\OneDrive\Documents\lollms\installer_files$$, mais le problème persiste.

Pourriez-vous me fournir des instructions détaillées sur la façon de résoudre ce problème ? Y a-t-il des étapes spécifiques que je devrais suivre pour assurer une installation réussie de Lollms ?

Je vous remercie d'avance pour votre aide.

Cordialement,
[
![Capture](https://github.com/user-attachments/assets/de9f5be3-9028-41ce-a781-5613f4a3527f)

Citations:
[1] https://ppl-ai-file-upload.s3.amazonaws.com/web/direct-files/17231052/88beefd0-5e6b-475a-bdf8-faa95f368fb7/Capture.pdf
[2] https://ppl-ai-file-upload.s3.amazonaws.com/web/direct-files/17231052/07e665df-1d52-4eef-88fe-1e409c627abf/Capture-ANACONDA.pdf
[3] https://ppl-ai-file-upload.s3.amazonaws.com/web/direct-files/17231052/b5808029-7b9a-44f0-9bdc-8e80484fe87d/Lollms.pdf

**Comment by ParisNeo** [comment]
Bonjour,

êtes vous sur que vous avez mis l'installeur dans un dossier ne contenant pas d'espace comme précifié dans le message qui s'affiche au début de l'installation?

**Comment by zokalinga** [comment]
Bonjour, 
je ne sais pas franchement car je ne suis pas très doué en informatique.
